src/engine/x86-64/X86_64SinglePassCompiler.v3

```diff
@@ -564,6 +564,11 @@ class X86_64SinglePassCompiler extends SinglePassCompiler {
 	def visit_F64X2_LT() { do_op2_x_x(ValueKind.V128, asm.cmpltpd_s_s); }
 	def visit_F64X2_GE() { do_c_op2_x_x(ValueKind.V128, asm.cmplepd_s_s); }
 	def visit_F64X2_LE() { do_op2_x_x(ValueKind.V128, asm.cmplepd_s_s); }
+	def visit_F64X2_PMIN() { do_c_op2_x_x(ValueKind.V128, asm.minpd_s_s); }
+	def visit_F64X2_PMAX() { do_c_op2_x_x(ValueKind.V128, asm.maxpd_s_s); }
+	def visit_F64X2_MIN() { do_op2_x_x(ValueKind.V128, mmasm.emit_f64x2_min(_, _, X(allocTmp(ValueKind.V128)))); }
+	def visit_F64X2_MAX() { do_op2_x_x(ValueKind.V128, mmasm.emit_f64x2_max(_, _, X(allocTmp(ValueKind.V128)))); }
```

Comment:
There are many of these 2-input instructions that now use a temp. We can consider factoring them out into a helper in a future CL.